base/math.jl

```diff
@@ -947,11 +947,15 @@ end
 @inline function ^(x::Float64, y::Float64)
```

Comment:
This `@inline` seems potentially problematic now that this function is no longer small.